### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by ToddDaniel** [jasp-stats/jasp-issues#2205]
### Description

Distributions allow for more decimal places

### Purpose

Teaching students about proportions under the normal curve using the Distributions module. For example, “what proportion of scores is above z = +1.23?” In an Excel example, we get .1093 but JASP reports only two decimals: .11. Although this is accurate, the homework often calls for more than two decimals. Setting the decimals to a default of 4 does not change the value reported on the probability distribution image. 

### Is your feature request related to a problem?

Not a JASP problem but a pedagogical problem. Excel gives me 4 decimals for probability and I want to align the JASP teaching to the Excel example

### Is your feature request related to a JASP module?

Distributions

### Describe the solution you would like

When I set the decimals to 4 in the preferences, I would like the results in the probability  distribution to also show 4. Currently they show two decimals. Even better would be a separate one of output that reported the probability to the default number of decimals, that way you would not need to address the curve, just add a result

### Describe alternatives that you have considered

I used right click and double click trying to find an option for adjusting decimals

### Additional context

**Steps Teaching this Technique - from the tutorial I made for my students:**
Open JASP
Show Modules Menu -> Distributions
The Show Modules Menu is the blue plus sign (+) in the upper right corner
**Set Decimals to 4**
Select Main Menu Icon -> Preferences -> Results. Under the Table Options use the plus and minus signs to Fix the number of decimals to 4.
**Normal Distribution in JASP**
Click on Distributions menu in the ribbon, choose Normal
Under Parameters select μ,σ ; Leave Mean set to 0 and SD to 1
Under Options, Highlight select Probability; Interval is now active
Use the settings to find intervals, above or below infinity. <examples from our stats course follow>

**Comment by tomtomme** [comment]
Yeah would be nice, if the module would respect the Preferences -> Results > decimal setting.
I think 2 decimals are fine in most situations, but sometimes I like 3. 4 would be too much from my pedagogical perspective. 
Being flexible via the settings would be the best.

**Comment by boutinb** [comment]
The decimal setting is used automatically for tables (the analysis does not have to bother about that). In a plot, this must be I think programmatically added. @Kucharssim ?

**Comment by Kucharssim** [comment]
Yes, we can increase the decimals in the plots, but as @tomtomme says the optimal number may depend on the use of the analysis.

As the number of decimals setting is now forwarded to R (https://github.com/jasp-stats/INTERNAL-jasp/issues/2172), we can do what @tomtomme suggests and let the number of decimals be determined in the preferences, instead of hard coded in the analysis.

**Comment by tomtomme** [comment]
still valid for 0.18.1

**Comment by tomtomme** [comment]
@ToddDaniel 
Could you edit the title to reflect the goal of this request?

**Comment by tomtomme** [comment]
@Kucharssim 
I guess this is now something for 0.20?

**Comment by tomtomme** [comment]
closing as duplicate of https://github.com/jasp-stats/jasp-issues/issues/1270
